language-support/qute/qute.tmLanguage.json

```diff
@@ -198,15 +272,19 @@
         }
       },
       "patterns": [
+        {
+          "match": "(or|orEmpty)",
+          "name": "keyword.control.qute"
+        },
+        {
+          "match": "(&&|\\|\\||\\?:|\\?|:)",
```

Comment:
Is | is allowed?

Reply:
The | is the | to match, it comes after the escape of ||, which is \\|\\|

Reply:
If I were to match | in the regex, it would be something like `\\|`, so the full match would be `(&&|\\||\\|\\||\\?:|\\?|:)`, where the | comes after &&

Reply:
Ok thanks for your explanation.